### ISSUE TITLE
Community icon contribution guideline request

### ISSUE THREAD

**Issue by samsam07** [CoreyGinnivan/system-uicons#16]
This is a superb project and I believe many people like myself would like to contribute and grow the icon base for everyone.
Will it be possible to provide some guidelines about the design language so as to maintain consistent icons submitted by different future contributors?

I have a few icons I believe could be useful for the project.
Anyway, thanks for providing a completely free way of using these icons however we want.

**Comment by CoreyGinnivan** [comment]
Yep can do! I'm writing up a blog post detailing the design decisions, but that should be easy enough to convert it to guidelines :)

**Comment by mrtnzlml** [comment]
> I'm writing up a blog post detailing the design decisions

Hi @CoreyGinnivan! Just wanted to drop a question about this blog post - did you already publish it somewhere? 😎
